_posts/2020-07-06-whats-so-super-about-super-tdma.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+comments: true
+title: What's so "super" about Super-TDMA ?
```

Comment:
While the title is _catchy_, might we benefit from a mention of UnetStack being able to do it somehow? To think about...

Reply:
The title I used went very well with the flow of this blog in my opinion but I agree that mention of UnetStack may help. I have these three titles in mind. Please do suggest your preference on these below-mentioned titles or let me know if you have think of a different title.

1. What's so "super" about Super-TDMA ?
2. Simulating protocols exploiting long propagation delays in UnetSim
3. Super-TDMA : Key concepts, practical challenges and it's simulation in UnetSim

Reply:
1 still is best :-)

Reply:
Ok :) I will stick with this title then.